vertx-web/src/main/java/io/vertx/ext/web/templ/impl/CachingTemplateEngine.java

```diff
@@ -42,6 +42,9 @@ protected String adjustLocation(String location) {
     if (!location.endsWith(extension)) {
       location += extension;
     }
+    if (location.endsWith("/" + extension)) {
```

Comment:
not really sure we want this, or at least we need some discussion on how to proceed.